webrtcdirect.go

```diff
@@ -9,7 +9,6 @@ import (
 
 var log = logging.Logger("webrtcdirect-tpt")
 
-var webrtcma, _ = ma.NewMultiaddr("/p2p-webrtc-direct")
```

Comment:
Because the new Multiaddress format requires a peer id after `p2p-webrtc`, we can't create it statically anymore.